schema/360-giving-package-schema.json

```diff
@@ -17,6 +17,15 @@
           "type": "string",
           "pattern": "^(\\d+\\.)(\\d+)$"
         },
+	"extensions": {
+	  "title": "Extensions",
+	  "description": "An array of strings which refer to Extensions in use by this 360Giving data file. Please see the Guidance for details.",
```

Comment:
This text was included in the proposal, and serves its purpose. We will however need to do a PATCH update this to include a link to the Extensions guidance when that is ready - which may be after the point that we want this update to be made live.

If we have an opportunity to review the description text, I suggest this rewording

"An array of strings which refer to 360Giving Extensions in use by this data file. See the 360Giving Extensions guidance for details." [360Giving Extensions guidance] would be hyperlinked.

Reply:
I've pushed a commit amending the schema. I've included the hyperlink but pointed it  at the Metadata Field guidance temporarily.

If this would be too confusing, I'm happy to remove the hyperlink until we've prepared the specific Extension Guidance and we've got a better target for the hyperlink.